docs/parts/prism_central.md

```diff
@@ -0,0 +1,108 @@
+<!-- See https://squidfunk.github.io/mkdocs-material/reference/ -->
+
+# Part 5: Deploy Prism Central
+
+Many of the more powerful capabilities of Nutanix are only unlocked after configuring Prism Central. In this part of the workshop, we will deploy Prism Central on our Nutanix cluster.
```

Comment:
Unless there are some restrictions with us adding external links, I would refer to the Prism Central documentation. I'm not sure if it's a better link I just found that one

```suggestion
Many of the more powerful capabilities of Nutanix are only unlocked after configuring [Prism Central](https://portal.nutanix.com/page/documents/details?targetId=Prism-Central-Guide-vpc_2023_4:mul-pc-overview-c.html). In this part of the workshop, we will deploy Prism Central on our Nutanix cluster.
```